src/cloudai/report_generator/training/parser.py

```
@@ -74,11 +74,16 @@ def get_model_name(self, tr: TestRun) -> str:
 
     def can_parse(self, tr: TestRun) -> bool:
         """Return True when the run produced the TB events and config artifact this parser needs."""
+        name = type(self).__name__
         tb_dir = self.get_tb_dir(tr)
         if not (tb_dir.is_dir() and any(tb_dir.iterdir())):
+            logging.warning(f"{name}: no TensorBoard events at '{tb_dir}'; skipping training report")
             return False
         config_path = self.get_config_path(tr)
-        return config_path is not None and config_path.is_file()
+        if config_path is None or not config_path.is_file():
+            logging.warning(f"{name}: config artifact not found under '{tr.output_path}'; skipping training report")
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)